packages/create-svelte/template/jsconfig.json

```diff
@@ -1,5 +1,8 @@
 {
 	"compilerOptions": {
+		"baseUrl": ".",
+		"checkJs": true,
+		"types": ["vite/client"],
```

Comment:
This should be solved in a different way. Adding types closes off all other ambient types in the project. People would need to add stuff like `"jest"` which is a source of confusion. I think the same can be achieved with triple slash references in the `globals.d.ts`.

Reply:
Oh sweet, that works! Will do!